lib/thirdparty/Asana.class.php

```diff
@@ -14,14 +14,17 @@ public static function listRoadmapTasks($cache = true)
       161514803479899 => ['Blockchain and Wallets', 'https://github.com/lbryio/lbrycrd'],
       136290697597644 => ['Integration and Building', null],
       158602294500249 => ['Documentation', null],
+      658477315612493 => ['Complete', null],
+      658477315612495 => ['In progress', null]
     ];
 
     $tasks = [];
 
     $tags = [
       192699565737944 => 'Open Beta',
       542803886522122 => 'Upcoming',
-      542803886522120 => 'Future'
+      658477315612491 => ' 2018 ',
```

Comment:
Had to leave a space to prevent php from converting to int and changing it to zero. Even using (string) has not fixing it.